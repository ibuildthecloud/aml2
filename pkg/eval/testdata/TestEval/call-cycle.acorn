f: function {
	return: f()
}
b: f()