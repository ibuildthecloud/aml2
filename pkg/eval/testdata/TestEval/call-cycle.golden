"exceed max call depth 101 > 100: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:2:11: call-cycle.acorn:4:5"
